dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -285,6 +285,46 @@ void Presolver<bias_type, index_type, assignment_type>::apply() {
             changes = remove_zero_biases(model_.constraint_ref(c)) || changes;
         }
 
+        // *-- clear out small linear biases in the constraints
+        // todo: temporarily hard-coded the FEASIBILITY_TOLERANCE here.
+        // todo: ideally this should be added to dimod.
+        bias_type FEASIBILITY_TOLERANCE = 1.0e-6;
+        for (size_type c = 0; c < model_.num_constraints(); ++c) {
+            auto& constraint = model_.constraint_ref(c);
+            std::vector<index_type> small_biases;
+            std::vector<index_type> small_biases_temp;
+            bias_type reduction = 0;
+            bias_type reduction_limit = 0;
+
+            for (auto& v : constraint.variables()) {
+                // ax ◯ c
+                bias_type a = constraint.linear(v);
+                bias_type lb = constraint.lower_bound(v);
+                bias_type ub = constraint.upper_bound(v);
+                bias_type v_range = ub - lb;
+
+                if (abs(a) < 1.0e-3 && abs(a) * v_range * constraint.num_variables()
```

Comment:
Need `#include <cmath>` and should use `std::abs(...)`.

Reply:
done